server/sonar-web/src/main/js/apps/tutorials/onboarding/ProjectKeyStep.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+// @flow
+import React from 'react';
+import { createProject, deleteProject } from '../../../api/components';
+import { translate } from '../../../helpers/l10n';
+
+type Props = {
+  onDelete: () => void,
+  onDone: (projectKey: string) => void,
+  organization?: string,
+  projectKey?: string
+};
+
+type State = {
+  done: boolean,
+  loading: boolean,
+  projectKey: string
+};
+
+export default class ProjectKeyStep extends React.PureComponent {
```

Comment:
It's very similar to the `NewOrganizationForm` component, it could be nice to have similar component names like `NewProjectForm` (it's really creating a project too, not only choosing a project key).